docs/index.html

```diff
@@ -245,6 +245,93 @@ <h2 id="questions-and-issues">Questions and issues</h2>
+
+<p>But scale how? More specifically, MAGE helps you:</p>
+
+<ul>
+<li>Scale your code; MAGE helps you write code that fits in your head</li>
+<li>Scale your runtime; your game server will run on your laptop, in
+a large cluster, and on anything in between</li>
+<li>Scale your ops; MAGE provides all the necessary APIs for you to be
+able to monitor your game server</li>
+</ul>
+
+<h3 id="rich-ecosystem">Rich ecosystem</h3>
+
+<p>MAGE provides only the primitives required to write interactive games. However,
+MAGE comes with a rich ecosystem of official and third party modules that will help
+you add features such as static data management, maintenance management, and so on.</p>
+
+<p>Official MAGE modules can be found on <a href="https://github.com/mage">GitHub</a>.</p>
+
+<h2 id="features">Features</h2>
+
+<h3 id="typescript-javascript">TypeScript &amp; JavaScript</h3>
+
+<p>MAGE actively supports both JavaScript and TypeScript projects; you can easily choose in which language you wish to create a project, and many tools support both languages.</p>
+
+<h3 id="transactional-api-endpoints">Transactional API endpoints</h3>
+
+<p>This is the core feature of MAGE; it provides you with a
+<a href="#states">State API</a> that helps you create transactional API endpoints.</p>
+
+<p>Transactional API endpoints (named <a href="#user-commands">user commands</a>)
+will allow you to stack all your data mutations and asynchronous messages
+on a state object. This state object will either automatically be commited
+once your API call completes, or rollback if an error occurs.</p>
+
+<h3 id="multiple-storage-backends">Multiple storage backends</h3>
+
+<p>The <a href="#archivist">data storage API</a> allows you to abstract your storage
+backend. Not only it will allow you to access and store data to your
+database of choice, but it will also help you with database migrations
+whenever needed.</p>
+
+<h3 id="built-in-distributed-mode">Built-in distributed mode</h3>
+
+<p>Thanks to the built-in <a href="api.html#service-discovery">service discovery system</a>,
+all you need to do to deploy your game server in a cluster is configure
+the discovery service.</p>
+
+<p>This means that your messages will always be routed to the right server and
+correctly forwarded to the right player.</p>
+
+<h3 id="rich-ecosystem-of-sdks-modules-and-tools">Rich ecosystem of SDKs, modules and tools</h3>
+
+<p>MAGE officially provides client SDKs for both
+<a href="https://github.com/mage/mage-sdk-js">HTML5</a> and
+<a href="https://github.com/mage/mage-sdk-unity">Unity</a> games,
+making it easier than ever to connect your game to your servers.</p>
+
+<p>On top of that, the <a href="https://github.com/mage">mage organization on GitHub</a>
+hosts a wide range of additional tools and modules which can help you
+with various aspects of your MAGE development pipeline.</p>
+
           <h1 id="requirements">Requirements</h1>
```

Comment:
That would be great indeed, and as I have mentioned before, I believe the best way to post this kind of content would be on the blog.

What I could do for the moment is link to the official blog as well. Would that make sense to you?